### DISCUSSION TITLE
Right way to load third-party/external scripts?

### DISCUSSION THREAD

**Discussion by dmahajan980** [shuding/nextra#1496] [Q&A]
Hello, I'm new to Nextra and MDX-driven concept of websites. I could not find any information on the right way to load third-party scripts in Nextra. Can you please explain briefly on how to do so?

Thanks

**Comment by fuma-nama** [comment]
Nextra doesn’t give any opinions about adding 3rd party scripts

You can use [Custom App](https://nextjs.org/docs/advanced-features/custom-app), and [next/script](https://nextjs.org/docs/api-reference/next/script) for that
